server/services/prisonerService.js

```diff
@@ -114,25 +123,28 @@ function createPrisonerService(nomisClientBuilder) {
     }
   }
 
-  async function getRoDetails(nomisClient, roRelation) {
-    const { personId } = roRelation
+  async function getRoDetails(nomisClient, roPerson) {
+    const { personId } = roPerson
     const personIdentifiers = await getPersonIdentifiers(nomisClient, personId)
 
     if (!personIdentifiers || personIdentifiers.length < 1) {
-      throw new RoRelationshipError('No person identifiers')
+      logger.warn(`No person identifiers for RO person: ${roPerson}`)
+      return roPerson
```

Comment:
we're checking each person id now, so we don't thrown errors here because that would stop us checking the next ones